pages/user-profile/[id]/index.js

```diff
@@ -0,0 +1,203 @@
+import { FaBriefcase } from "react-icons/fa";
+import { AiFillPhone } from "react-icons/ai";
+import { GoLocation } from "react-icons/go";
+import { AiOutlineMail } from "react-icons/ai";
```

Comment:
Since this page is almost the same as the company profile page, it's preferable to use the same icons to have a consistent design.